frontend/src/components/dialog/DialogPeriodForm.vue

```diff
@@ -9,12 +9,10 @@
       v-model="period.start"
       :name="$tc('entity.period.fields.start')"
       vee-id="start"
-      :value-format="$moment.ISO_8601"
       vee-rules="required" />
 
     <e-date-picker
       v-model="period.end"
-      :value-format="$moment.ISO_8601"
       :name="$tc('entity.period.fields.end')"
       vee-rules="required|minDate:@start" />
```

Comment:
```suggestion
      vee-rules="required" />
```
würde da jetzt noch ohne `mindDate` fahren, da das ja noch nicht in allen Sprachen funktioniert (#525) nicht fuktioniert